BoyumFoosballStats/Services/MatchMakingService.cs

```diff
@@ -24,6 +24,84 @@ public MatchMakingService(IAzureBlobStorageHelper blobStorageHelper)
         _blobStorageHelper = blobStorageHelper;
     }
 
+    public async Task<Match> AutoSwapPlayers(List<Match> matches, List<Player> players,
+        MatchMakingMethod? matchMakingMethod = null)
+    {
+        if (matches.Count == 0)
+        {
+            //If no matches are supplied - find the fairest match
+            return await FindFairestMatch(players, matchMakingMethod ?? MatchMakingMethod.Ai);
```

Comment:
Hmm, I'm not sure - I've been trying to cover all the bases, and I figured that if we have no prior matches or have to swap everyone, we might as well balance the match, as there is no other parameter to choose based on.

I would be fine just grabbing the first 4 people in either case - that being said, this scenario should never happen as it will always be triggered based on a match being saved. So there should always be a prior match to act upon.